operators/operator_sdk/osdk-scorecard.adoc

```diff
@@ -11,7 +11,7 @@ scorecard tool to validate your Operator packaging and run tests.
 
 [NOTE]
 ====
-{product-title} 4.4 supports Operator SDK v0.15.0.
+{product-title} 4.4 supports Operator SDK v0.19.4.
```

Comment:
@adellape "4.4 supports Operator SDK v0.19.4" should be "4.6 supports Operator SDK v0.19.4" because it is for 4.6 doc.